packages/paste-website/src/__tests__/genericHeader.test.tsx

```diff
@@ -0,0 +1,86 @@
+import * as React from 'react';
+import {Theme} from '@twilio-paste/theme';
+import {GenericHeader} from '../components/shortcodes/generic-header/index';
+import {render, screen} from '@testing-library/react';
+
+describe('GenericHeader', () => {
+  const FullHeader: React.FC = () => {
+    return (
+      <Theme.Provider theme="default">
+        <GenericHeader
+          name="Alert"
+          categoryRoute="/components"
+          description="This is an alert component."
+          designCommitteeReview={null}
+          engineerCommitteeReview={null}
+          figmaStatus={null}
+          githubUrl="https://google.com"
+          packageName="@twilio-paste/alert"
+          packageStatus="alpha"
+          storybookUrl="/?path=/story/components-alert--neutral"
+          version="1.0.0"
+        />
+      </Theme.Provider>
+    );
+  };
+  // These tests are more appropriate as cypress tests
+  // it('should add openGraph meta for existing components and primitives', () => {
+  //   const openGraphMeta = screen.getBy
+  // });
+
+  // it('should not add openGraph meta for non-existing components and primitives', () => {
+
+  // });
+
+  // it('should not add openGraph meta for non-components, non-primitives pages', () => {
+
+  // });
+
+  it('should render an h1 tag using the name prop as text', () => {
```

Comment:
Suggestion: you could combine all the "should render..." tests into one big test with a bunch of expect statements since they all render the same thing.

Reply:
True, and it'd be less repetitive, but I'd rather keep them separate because then the tests fail in a more granular way
<img width="638" alt="image" src="https://user-images.githubusercontent.com/77117846/180567434-a3356f2b-507c-4e7d-b767-fb4042beca16.png">
<img width="621" alt="image" src="https://user-images.githubusercontent.com/77117846/180567375-5e0d9f63-1301-408f-ab4d-3a60a22d45a4.png">